<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="SQLtab">
     <attribute name="title">
      <string>SQL-tab</string>
     </attribute>
     <widget class="QRadioButton" name="RBSQL1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBSQL2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBSQL3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 3</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TESQL1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>631</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTA QUE SE VA A REALIZAR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBSQL4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBSQL5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QUERYS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTA QUE SE VA A REALIZAR</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TESQL2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>721</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Slider">
     <attribute name="title">
      <string>IA</string>
     </attribute>
     <widget class="QPlainTextEdit" name="TEIA2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBIA2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QUERYS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTA QUE SE VA A REALIZAR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBIA1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTA QUE SE VA A REALIZAR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RBIA3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulta 3</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TEIA1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Foto</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>351</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>c1.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNFOTO1">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TEFOTO1">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>230</y>
        <width>104</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TEFOTO2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNFOTO2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>530</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CAMBIAR</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="TEFOTO3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>440</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
